server/protocols/jmap/src/main/java/org/apache/james/jmap/mailet/VacationMailet.java

```diff
@@ -0,0 +1,116 @@
+/****************************************************************
```

Comment:
you have an ordering problem : this class should be created before previous commit